pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -440,8 +438,8 @@ func (c *NetworkPolicyController) GetAssociatedIPBlockGroups(ip net.IP) []antrea
 	for _, obj := range ipBlockGroupObjs {
 		group := obj.(*antreatypes.Group)
 		for idx := range group.IPNets {
-			ipNet := &group.IPNets[idx]
-			if ipNet.Contains(ip) {
+			ipNet := group.IPNets[idx]
+			if ipNet != nil && ipNet.Contains(ip) {
```

Comment:
if `group.IPNets` stores pointers now, we can probably go back to the `for _, ipNet := range group.IPNets` loop